lib/src/main/java/org/team100/lib/motor/turning/Falcon6TurningMotor.java

```diff
@@ -26,49 +26,45 @@
  * See {@link FalconDriveMotor} for configuration details.
  */
 public class Falcon6TurningMotor implements Motor100<Angle100> {
-    private static final double ticksPerRevolution = 2048;
+    //TODO Tune ff for amps
     private final double m_gearRatio;
     private static final double kCurrentLimit = 40;
 
     /**
      * The speed, below which, static friction applies, in motor revolutions per
      * second.
      */
-    private static final double staticFrictionSpeedLimitRev_S = 1.0;
+    private static final double staticFrictionSpeedLimitRev_S = 3.5;
 
     /**
-     * Friction feedforward in volts, for when the mechanism is stopped, or nearly
+     * Friction feedforward in amps, for when the mechanism is stopped, or nearly
      * so.
      */
-    private static final double staticFrictionFFVolts = 0.375;
+    private static final double staticFrictionFFAmps = 0.375;
 
     /**
-     * Friction feedforward in volts, for when the mechanism is moving.
+     * Friction feedforward in amps, for when the mechanism is moving.
      */
-    private static final double dynamicFrictionFFVolts = 0.27;
+    private static final double dynamicFrictionFFAmps = 0.27;
 
     /**
-     * Velocity feedforward in units of volts per motor revolution per second, or
-     * volt-seconds per revolution. Since saturation is 11 volts and free speed is
+     * Velocity feedforward in units of amps per motor revolution per second, or
+     * amp-seconds per revolution. Since saturation is 11 amp and free speed is
```

Comment:
you can't just search and replace the word "volt" without changing the meaning of this sentence.